<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="hh" failures="2" tests="6" name="Testlistner1" time="0.236" errors="0" timestamp="2022-02-27T19:57:48 IST">
  <testcase classname="testNGListners.TestClass" name="test1" time="0.029"/>
  <testcase classname="testNGListners.TestClass" name="test2" time="0.016">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNGListners.TestClass.test2(TestClass.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="testNGListners.TestClass" name="test3" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [hemant] but found [HEMANT]">
      <![CDATA[java.lang.AssertionError: expected [hemant] but found [HEMANT]
at testNGListners.TestClass.test3(TestClass.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="testNGListners.TestClass" name="test6" time="0.01"/>
  <testcase classname="testNGListners.TestClass" name="test4" time="0.004"/>
  <testcase classname="testNGListners.TestClass" name="test5" time="0.001">
    <skipped/>
  </testcase> <!-- test5 -->
</testsuite> <!-- Testlistner1 -->
